build.ps1

```diff
@@ -70,6 +71,7 @@ Param(
     [string] $framework = "*",
     [switch] $skipIotHubTests,
     [switch] $skipDPSTests,
+    [switch] $skipTPMTests,
```

Comment:
How about we mark them as flaky, as their behavior is currently flaky.
You could add a comment explaining that TPM only accepts a single connection at a time, and we don't have a way to effectively ensure that only a single test will make a call to TPM since we have class-level test parallelization. This makes the test behavior non-deterministic and flaky.

Reply:
There is an existing test category called "Flaky"

Reply:
Makes sense, changed.